spec/requests/alert_definitions_spec.rb

```diff
@@ -71,7 +71,7 @@
     sample_alert_definition = {
       "description" => "Test Alert Definition",
       "db"          => "ContainerNode",
-      "expression"  => { "eval_method" => "dwh_generic", "mode" => "internal", "options" => {} },
+      "expression"  => {"exp" => {"=" => {"field" => "Vm-name", "value" => "foo"}}, "context_type" => nil},
```

Comment:
Just curious about these changes, description mentioned being backward compatible, why are these needed ?

Reply:
Because, as @imtayadeway noted in this comment https://github.com/ManageIQ/manageiq-api/pull/76#discussion_r154723513, the expression there was not actually a MiqExpression, so that means the test was wrong. It wasn't supposed to "work" like that.

Reply:
👍 thanks for clarifying.